examples/fastapi/main.py

```
@@ -0,0 +1,403 @@
+            """스트리밍 응답 생성기"""
+            try:
+                async for chunk in llm.ask_stream_async(
+                    input=request.message,
+                    system_prompt=request.system_prompt,
+                    temperature=request.temperature,
+                    max_tokens=request.max_tokens
+                ):
+                    if chunk.text:
+                        yield f"data: {chunk.text}\n\n"
+                
+                yield f"data: [DONE]\n\n"
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)